build-package.sh

```diff
@@ -1,26 +1,28 @@
 #!/bin/sh
 
-if [ -z "$1" ]
-    then
-        echo "Release version wasn't specified";
-        return;
-fi
+release_version=$(jq -r '.version' ./composer.json);
 
-release_version=$1;
+echo "Creating temporary directory"
+rm -rf ./tmp/admin;
```

Comment:
I think we clean the entire `./tmp` on each build

Reply:
Done